wordpress-mobile/WordPress-iOS
# Add new TimeZoneSuggester

Timezone selection from site settings in iOS app is outdated in comparison with WordPress web and Android. This PR brings iOS on par with them by adding a TimeZone Suggester feature.  See https://github.com/wordpress-mobile/WordPress-iOS/issues/11450

Ref #11450 

### To test:
**To test TimeZone Suggester feature:**
1.  Navigate to My Site -> Site Settings -> Time Zone

2.  Make sure the time zone that is suggested is consistent with the timezone of the device / simulator you are testing on

3.  Tap the suggested timezone

4.  Step 3 will navigate you back to Site Settings screen where you should see the Suggested time you have chosen

5.  Tap Time Zone again from Site Settings screen


**To test search results works normally and has no regression:**

6. Search for a timezone (example Casablanca).  TableView results should filter and show Casablanca.

7. Tap Casablanca

8. Step 7 will navigate you back to Site Settings screen where you should see the Suggested time you have chosen (Casablanca)

![image](https://user-images.githubusercontent.com/88816724/132994428-17915fed-6519-405c-9750-335cc736b792.png)





## Regression Notes
1. Potential unintended areas of impact
TimeZone selector screen was changed - unintended impact could be user would not be able to change their TimeZone

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Initial tests are manually tested.  Screenshots above

3. What automated tests I added (or what prevented me from doing so)
No automated tests existed previously and ViewController is not in a state that facilitates automated testing

PR submission checklist:
